enterprise/api/v1/permissions.py

```diff
@@ -0,0 +1,16 @@
+""" Custom API permissions. """
+
+from rest_framework import permissions
+
+
+class IsAdminUserOrInGroup(permissions.BasePermission):
```

Comment:
As is, this class is specific to the enterprise api group. I would recommend making this an abstract parent class, and having a class specific to the group that inherits from it. Then the constant for the enrollment api access group that is defined below can be moved into the permission class.

Reply:
@brittneyexline I am using a Rest-Framework class hierarchy in which our classes are inherit from permissions.BasePermission and provide the implementation of has_permission method. 
Making an abstract base class does not look good to me here. 
Simply, I would suggest something like that: 
```
class IsEnterpriseEnrollmentAPIAccessed(permissions.IsAdminUser):
    ENTERPRISE_API_ALLOWED_GROUPS = ['enterprise_enrollment_api_access', ]

    message = 'User is not allowed to access the view.'
    def has_permission(self, request, view, *args):
        return super(IsEnterpriseEnrollmentAPIAccessed, self).has_permission(request, view) or \
               request.user.groups.filter(name__in=self.ENTERPRISE_API_ALLOWED_GROUPS).exists()

```

Reply:
At a high level, it would be good to separate the common logic of check if this user is in a certain group away from checking for a specific group, so that it can be reused later. It doesn't have to be an abstract class, it could be a mixin or something else. All I care about is reusability of the group checking logic and naming the class responsible for this particular group's check in a way that is specific to that group.